Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Prepare the boot build image and dependencies
+FROM clojure:boot-alpine AS boot
+RUN apk add --update git npm
+RUN npm install -g yarn shadow-cljs
+
+# Copy source files into the docker container
+WORKDIR /app
+COPY . ./
+
+# Update game data to the latest release
+RUN git submodule update --init
+RUN cd resources/game \
+ && git checkout master \
+ && git checkout $(git describe --tags --abbrev=0 $(git rev-list --tags --max-count=2) | grep -vE 'continuous')
+
+# Build
+RUN boot build
```

Comment:
For some reason I keep getting `java.lang.OutOfMemoryError: Java heap space` here, no matter how much memory is allocated to docker or to JVM process (while outside docker it works just fine). This should be investigated.

Reply:
I got this error when trying to run (in docker or externally) against the current version of the game. If you restrict to v9.14 it should work?

Not sure why parser errors result in a memory leak, but that's always how they manifested for me until I made the fixes in  #13

Reply:
Oh, it's about parser errors, now it makes sense. Yes, there is a problem with this parser - sometimes when it can't parse something it goes into an infinite loop and seems to eat a lot of system resources.

I'll try to run it on a stable version.

Reply:
This turned out to be a problem with outdated Docker Desktop installation on my machine; seems like it restricted memory consumption to 2gb per container and didn't allow to change this limit, but when I updated it to 4.15.0, the limit went up to 8gb per container and now it successfully builds the app.